dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -341,27 +341,24 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
     // 7-day window allowed by the server-side function.
     ZonedIntervalIterable intervals = ZonedIntervalIterable.forConnectorArguments(arguments);
     LOG.info("Exporting query log for " + intervals);
-    for (ZonedInterval zonedInterval : intervals) {
-      TaskDescription taskDescription;
-      if (arguments.isAssessment()) {
-        taskDescription =
-            new TaskDescription(
-                QueryHistoryExtendedFormat.ZIP_ENTRY_PREFIX,
-                createExtendedQueryFromAccountUsage(arguments),
-                QueryHistoryExtendedFormat.Header.class);
-        addJdbcTask(out, zonedInterval, taskDescription);
-      } else {
-        taskDescription =
-            new TaskDescription(ZIP_ENTRY_PREFIX, newQueryFormat(arguments), Header.class);
-      }
-      addJdbcTask(out, zonedInterval, taskDescription);
-    }
 
-    if (arguments.isAssessment()) {
-      List<TaskDescription> tasks = createTaskDescriptions(arguments);
-      ZonedIntervalIterable.forConnectorArguments(arguments, ChronoUnit.DAYS)
-          .forEach(interval -> tasks.forEach(task -> addJdbcTask(out, interval, task)));
+    if (!arguments.isAssessment()) {
+      TaskDescription taskDescription =
+          new TaskDescription(ZIP_ENTRY_PREFIX, newQueryFormat(arguments), Header.class);
+      intervals.forEach(interval -> addJdbcTask(out, interval, taskDescription));
+      return;
     }
+
+    TaskDescription taskDescription =
+        new TaskDescription(
+            QueryHistoryExtendedFormat.ZIP_ENTRY_PREFIX,
+            createExtendedQueryFromAccountUsage(arguments),
+            QueryHistoryExtendedFormat.Header.class);
+    intervals.forEach(interval -> addJdbcTask(out, interval, taskDescription));
+
+    List<TaskDescription> tasks = createTaskDescriptions(arguments);
```

Comment:
nit: change all names - methods and variables -  associated with the time series tasks to include "TimeSeries" (or "timeSeries") in the variable name, so that it is clear why they are done separately.